### PR TITLE
Create _child_why_AnVIL_instructor.Rmd

### PR DESCRIPTION
**Pull request by KatherineCox**
Updating the "Why AnVIL"? section

- Moving to child doc so it can be borrowed in other books
- The previous version talked up the advantages of cloud computing (which is great), but didn't really give an answer as to why AnVIL specifically, vs. say, ~RStudio~PositCloud.  This version keeps most of the points from the old one, reorganized a bit, and then adds some AnVIL-specific stuff.

### CODE REVIEW

File: child/_child_why_AnVIL_instructor.Rmd [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+- **Version control**:  All students will be using the exact same versions of programs, running on the exact same operating system, and the majority of setup can be handled for the students ahead of time.  This greatly reduces time spent on setup and prevents unexpected run errors during the activities, limiting the amount of troubleshooting you have to do.  As a result, classes and activities will run more smoothly and predictably.
+- **Inclusivity**: Cloud-computing enables all students to participate in the activity without needing to provide computers with certain specifications.
+- **Experience**: Cloud-computing is becoming common in today’s bioinformatics workplaces, and the classroom is a low-stress and low-stakes opportunity for students to gain experience.
+
+AnVIL is purpose-built for genomic data science:
+
+- **Tools**: Through AnVIL, students can access a wide range of tools (Galaxy, RStudio/Bioconductor, Jupyter, as well as scripted workflows) all in one place, and can easily transition between them.
+- **Datasets**: AnVIL provides a large and growing collection of relevant datasets with standardized organization, for easier importing and processing
```
Comment (avahoffman) [inline child/_child_why_AnVIL_instructor.Rmd:12]:
```suggestion
- **Datasets**: AnVIL provides a large and growing collection of relevant datasets with standardized organization, for easier importing and processing.
```

**Review by avahoffman — APPROVED** [review]

**Comment by avahoffman** [comment]
I wonder if this is something that can go in AnVIL_Template too?

**Comment by avahoffman** [comment]
I like the idea of most stuff being in AnVIL Template, and definitely don't want to be hunting around for commonly used content!

**Comment by KatherineCox** [comment]
Moving this to AnVIL_Template, https://github.com/jhudsl/AnVIL_Template/pull/189

**Pull request closed without merging**